### PR TITLE
Update search blank state

### PR DESCRIPTION
**Pull request by nicolaskempf57**
Close https://github.com/etalab/data.gouv.fr/issues/1099

### CODE REVIEW

File: udata_front/theme/gouvfr/assets/js/components/search/search.vue [hunk 1]
```diff
@@ -173,20 +203,22 @@
...
 import SchemaFilter from "./schema-filter.vue";
-import Empty from "./empty.vue";
 import { Pagination } from "@etalab/udata-front-plugins-helper";
 import MultiSelect from "./multi-select.vue";
-import { search_autocomplete_debounce } from "../../config";
+import ActionCard from "../Form/ActionCard/ActionCard.vue";
+import { data_search_form_url, search_autocomplete_debounce } from "../../config";
 import { debounce } from "../../composables/useDebouncedRef";
+import searchIcon from "svg/illustrations/search.svg";
+import search2Icon from "svg/illustrations/search2.svg";
```
Comment (maudetes) [inline udata_front/theme/gouvfr/assets/js/components/search/search.vue:221]: Ahah I wonder if we could have more explicit names for theses? Maybe the second one could be named `magnifying_glass.svg`?

File: udata_front/theme/gouvfr/assets/js/dsfr.js [hunk 2]
```diff
@@ -6,3 +6,41 @@
...
+
+/**
+ * This function returns the dsfr alignment to use for flex main axis alignment.
+ *
+ * @remarks
+ * It abstracts the fact that the dsfr name is made for flex-direction row and is odd for flex-direction column.
+ *
+ * @param {import("./types").AxisAlignment} alignment
+ */
+export function getMainAxisAlignment(alignment) {
```
Comment (maudetes) [inline udata_front/theme/gouvfr/assets/js/dsfr.js:18]: It's not needed in this PR right, but is used in publishing form?

File: udata_front/theme/gouvfr/assets/js/components/search/search.vue [hunk 3]
```diff
@@ -154,16 +154,46 @@
...
+                :title="$t(`Didn't find what you are looking for ?`)"
+                :icon="search2Icon"
+                type="primary"
+                >
+                <p class="fr-m-0 fr-text--sm">
+                  {{ $t("Try to reset filters to widen your search.") }}<br/>
+                  {{ $t("You can also give us more details with our feedback form.") }}
+                </p>
+                  <template v-slot:actions>
+                    <button @click="reloadForm" class="fr-btn fr-btn--secondary">
```
Comment (maudetes) [inline udata_front/theme/gouvfr/assets/js/components/search/search.vue:168]: Somehow when clicking the `reloadForm`, the URL's params get updated with weird `isTrusted=true&_vts=XXXXx`?

File: udata_front/settings.py [hunk 4]
```diff
@@ -20,6 +20,9 @@
 BANNER_HTML_CONTENT_EN = ''
 BANNER_HTML_CONTENT_FR = ''
 
+# Frontend data search form url
+DATA_SEARCH_FORM_URL = 'https://tally.so/r/mDKv1N'
```
[the remark this answers is not in the document]
Comment (maudetes) [reply udata_front/settings.py:24]: Maybe we could name it `DATA_SEARCH_FEEDBACK_FORM_URL`? To make the feedback form more obvious in the setting.

**Review by maudetes — COMMENTED** [review]
Nice rendering. I find it nice to add the search form with few results :+1:

File: udata_front/theme/gouvfr/assets/js/dsfr.js (hunk 2 above)
Comment (nicolaskempf57) [reply udata_front/theme/gouvfr/assets/js/dsfr.js:18]: It's used in the ActionCard component to allow to choose the buttons alignement. 
Here the value isn't provided so it defaults to "start".

Comment (nicolaskempf57) [reply udata_front/theme/gouvfr/assets/js/dsfr.js:18]: In the publishing form, the `end` alignment is used instead.

**Review by maudetes — APPROVED** [review]
:pray:

**Pull request merged**